pkg/scheduler/framework/framework.go

```diff
@@ -139,7 +138,7 @@ func (f *framework) VolumeBinder() scheduling.SchedulerVolumeBinder {
 }
 
 func NewFrameworkHandle(pods []*v1.Pod, nodes []*v1.Node) v1alpha1.FrameworkHandle {
-	snapshot := k8scache.NewSnapshot(pods, nodes)
+	snapshot := cache.NewSnapshot(pods, nodes)
```

Comment:
let's open an issue in upstream to make cache public.

Reply:
https://github.com/kubernetes/kubernetes/issues/91983

Reply:
ok, let's move this PR forward; I don-t think we will have a conclusion in a shot time :(

Reply:
ok, let's move this PR forward; I don-t think we will have a conclusion in a shot time :(